cadence/contracts/NFTCatalog.cdc

```diff
@@ -75,17 +71,17 @@ pub contract NFTCatalog {
     // NFTCatalogProposalManager
     // Used to authenticate proposals made to the catalog
 
-    pub resource interface NFTCatalogProposalManagerPublic {
-        pub fun getCurrentProposalEntry(): String?
+    access(all) resource interface NFTCatalogProposalManagerPublic {
+        access(all) fun getCurrentProposalEntry(): String?
     }
-    pub resource NFTCatalogProposalManager : NFTCatalogProposalManagerPublic {
+    access(all) resource NFTCatalogProposalManager : NFTCatalogProposalManagerPublic {
             access(self) var currentProposalEntry: String?
 
-            pub fun getCurrentProposalEntry(): String? {
+            access(all) fun getCurrentProposalEntry(): String? {
                 return self.currentProposalEntry
             }
 
-            pub fun setCurrentProposalEntry(identifier: String?) {
+            access(all) fun setCurrentProposalEntry(identifier: String?) {
```

Comment:
Should this method have an entitlement? It looks like it should only be callable by an admin